pageserver/src/tenant.rs

```diff
@@ -1698,6 +1700,11 @@ impl Tenant {
                     return Err(CreateTimelineError::AncestorNotActive);
                 }
 
+                if ancestor_timeline.is_archived() == Some(true) {
+                    info!("tried to branch archived timeline");
```

Comment:
warning?

Reply:
This is something that can easily be triggered by user mistake. I'd reserve warnings for bad situations that arise from internal pageserver behaviour, something that is either a bug or indicates a problem with the server (evicting too much due to full disk, etc).